storey/table.py

```diff
@@ -186,18 +192,21 @@ def _aggregates_to_schema(self):
         return schema
 
     def _get_aggregations_attrs(self, key):
+        key = get_hashed_key(key)
         if key in self._attrs_cache:
             return self._attrs_cache[key].aggregations
         else:
             return None
 
     def _set_aggregations_attrs(self, key, element):
+        key = get_hashed_key(key)
         if key in self._attrs_cache:
             self._attrs_cache[key].aggregations = element
         else:
             self._attrs_cache[key] = _CacheElement({}, element)
 
     def _get_static_attrs(self, key):
+        key = get_hashed_key(key)
```

Comment:
i can't put the comment in the right place but need to add this line in _set_static_attrs as well